src/Client.php

```diff
@@ -138,13 +138,13 @@ public function url()
     public function domain()
     {
     	if($this->sandboxMode) {
-    		$servers = array_only($this->config['server'], 'sandbox');
+    		$servers = Arr::only($this->config['server'], 'sandbox');
```

Comment:
Should we define the `Arr` class in the use section?

Reply:
Will you also add the `laravel/helpers` in the dependecies?

Reply:
> Will you also add the laravel/helpers in the dependecies?

Not sure what you mean here? are you saying keep the helpers and add the dependencies? what is the reason to add that dependency? I can do it if you want but not sure I understand what it will do?